### ISSUE TITLE
Iterables repeat parent element

### ISSUE THREAD

**Issue by mattstibbs** [delfick/python-dict2xml#8]
When converting the following:
```python
{
    'array_items': [
        { 
            'item': {
                'string1': 'string',
                'string2': 'string'
            }
        },{ 
            'item': {
                'string1': 'other string',
                'string2': 'other string'
        }
    ]
}
```
the xml output is:

```xml
<array_items>
    <item>
        <string1>string</string1>
        <string2>string</string2>
    </item>
</array_items>
<array_items>
    <item>
        <string1>other string</string1>
        <string2>other string</string2>
    </item>
</array_items>
```

Whereas I was expecting an output of:

```xml
<array_items>
    <item>
        <string1>string</string1>
        <string2>string</string2>
    </item>
    <item>
        <string1>other string</string1>
        <string2>other string</string2>
    </item>
</array_items>
```

Is it possible to output a list under the single parent? 

If not, would this be a useful option to have?

**Comment by delfick** [comment]
Hmm, that shouldn't be hard to make possible :)
*starts madly typing away*

**Comment by delfick** [comment]
I've released this under version 1.4 :)

**Comment by mattstibbs** [comment]
:smile:
